packages/core/src/core.ts

```diff
@@ -1,10 +1,12 @@
 import {issue, issueCommand} from './command'
-import {issueCommand as issueFileCommand} from './file-command'
+import {
+  issueCommand as issueFileCommand,
+  prepareKeyValueMessage as prepareCommandValue
```

Comment:
super nit but I find it easier to keep names consistent rather then renaming them across files just for search-ability later one

Reply:
I've renamed `issueCommand -> issueFileCommand` at the source too to avoid confusion with the existing `issueCommand`, what do you think?